src/ui/components/LedgerAdmin.js

```diff
@@ -137,12 +166,12 @@ export const LedgerAdmin = (): ReactNode => {
         />
         {selectedId && (
           <FormControlLabel
-            fullWidth
+            fullwidth="true"
```

Comment:
not sure why you added the string "true" on line 159 and 169. This technically works since the "true" string is not falsey, but the previous way of just setting the key as a Component property is more correct, since it assigns a boolean. Otherwise this branch looks good.

Reply:
@crisog

Reply:
Yea I changed the property to `fullwidth` because of a React warning, but after checking MaterialUI it seems that the correct way to do it was just `fullWidth`. About the string, yea that should not be there in any of the cases.